examples/meta/generator/generate.py

```diff
@@ -88,8 +88,13 @@ def translateExamples(inputDir, outputDir, targetsDir, ctagsFile,
             extension = target["FileExtension"]
 
             # Create directory if it does not exist
-            if not os.path.exists(directory):
-                os.makedirs(directory)
+            try:
+                os.makedirs(name=directory, exist_ok=True)
```

Comment:
In the rebased commit I've changed it to `except TypeError:` to make it clearer.